src/textual/geometry.py

```diff
@@ -307,21 +370,28 @@ def contains_region(self, other: Region) -> bool:
         )
 
     def translate(self, x: int = 0, y: int = 0) -> Region:
-        """Move the origin of the Region.
+        """Return a new region of the same size, with the origin shifted.
 
         Args:
-            translate_x (int): Value to add to x coordinate.
-            translate_y (int): Value to add to y coordinate.
+            x (int): Value to add to x-coordinate for the new region. Optional; defaults to 0.
+            y (int): Value to add to y-coordinate for the new region. Optional; defaults to 0.
 
         Returns:
-            Region: A new region shifted by x, y
+            Region: A new `Region` instance of the same size, shifted by (x, y).
         """
 
         self_x, self_y, width, height = self
-        return Region(self_x + x, self_y + y, width, height)
+        return Region((self_x + x), (self_y + y), width, height)
 
     def __contains__(self, other: Any) -> bool:
-        """Check if a point is in this region."""
+        """Check if a point is in this region.
+        
+        The return value is always False unless `other`
```

Comment:
@willmcgugan what do you think of this revised wording? I feel like I'm having trouble describing the relationship between the input type and the return type in the docstring. I tried adding some `overload`s but couldn't find a way to make MyPy happy about them.